Add tools WG (#33)
chipsalliance/tsc: working_groups/tools/README.md

# Tools Working Group

> The Tools working group of CHIPS Alliance covers the wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses around A/MS). 

Reviewer: Is the slash in "A/MS" commonly used?
```suggestion
The Tools working group (WG) of CHIPS Alliance covers a wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses on AMS design flows). 
```

## Mission

> The mission of the Tools WG will be to:

Reviewer: This section, the heart of the document, does a good job of covering our goals.  I'm trying to think it there is anything else we should be trying to achieve, but I think this covers our activities pretty well!  Nice job.

Author: Thanks!

* identify open source tools in the ASIC / FPGA domain and help them collaborate and interoperate
* share and co-develop new, open source based ASIC / FPGA design methodologies
* discuss and share experiences related to digital ASIC / FPGA design to identify gaps in the existing ecosystem and potential improvements

> * identify neighboring areas where open source ASIC / FPGA design methodologies could result in radical improvements (e.g. chiplets, memory designs etc.)

Reviewer: Minor clarification:
```suggestion
* identify neighboring areas where the application of open source ASIC / FPGA design methodologies could result in radical improvements (e.g. chiplets, memory designs etc.)
```

> * identify ways to use open source tools in production-level, marketable solutions, raise awareness of success stories

Reviewer: Supremely minor suggestion:
```suggestion
* identify ways to use open source tools in production-level, marketable solutions, and raise awareness of success stories
```
